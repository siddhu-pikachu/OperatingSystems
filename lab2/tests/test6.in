# ./(executable_name) -S test6.in

3 4
1,2,3,4,
5,6,7,8,
9,10,11,12,

3 4
1,4,3,4,
9,8,7,8,
3,10,4,12,
