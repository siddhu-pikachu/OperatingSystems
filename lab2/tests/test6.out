0,-2,0,0,
-4,-2,0,0,
6,0,7,0,
